web-client/src/AppInstanceManager.tsx

```diff
@@ -36,8 +38,13 @@ export const AppInstanceManager = connect(
         case 'stayLoggedIn':
           confirmStayLoggedInSequence();
           break;
-        case 'logout':
-          signOutSequence({
+        case 'idleLogout':
+          signOutIdleSequence({
+            skipBroadcast: true,
+          });
+          break;
```

Comment:
This separate message helps keep behavior between "regular sign out" and "idle sign out" distinct.